x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -282,6 +282,28 @@ str1:keyword |str2:keyword |ip1:ip  |ip2:ip
 // end::to_ip-result[]
 ;
 
+cdirMatchOrs
+required_capability: combine_disjunctive_cidrmatches
```

Comment:
Why do we need a capability here? Isn't this a performance improvement? So it should work in multi-version (?)

Reply:
> Why do we need a capability here? Isn't this a performance improvement? So it should work with old versions (?)

Let me double check this. Even thought the results should not change before and after this, the warnings might be different from main, there are warning for multi-valued field that CIDRMatch does not handle yet.